src/simcore_service_deployment_agent/auto_deploy_task.py

```diff
@@ -348,20 +368,23 @@ async def auto_deploy(app: web.Application):
         app["state"][TASK_NAME] = State.STOPPED
         return
     except Exception:  # pylint: disable=broad-except
-        log.exception("Error while initializing deployment: ", exc_info=True)
+        log.exception("Error while initializing deployment: ")
```

Comment:
I dont understand the reason for this change, the error Message was removed from the logs? 🧐 Could you please comment?

Reply:
``log.exceptions`` already include that flag.